hs_tools_resource/utils.py

```diff
@@ -53,3 +54,13 @@ def get_SupportedSharingStatus_choices():
             ['Discoverable', 'Discoverable'],
             ['Private', 'Private'],
             ]
+
+
+def convert_size(size_bytes):
+    if size_bytes == 0:
+        return "0B"
+    size_name = ("B", "KB", "MB", "GB", "TB")
```

Comment:
Planning to support TB scale images?

Reply:
:) why not? :)

No in reality, I thought this utility function might get used in other places that could have TB sized files...